source/remoteClient/dialogs.py

```diff
@@ -220,122 +280,114 @@ def onGetIPFail(self, exc: Exception) -> None:
 		)
 
 
-class DirectConnectDialog(wx.Dialog):
-	clientOrServer: wx.RadioBox
-	connectionType: wx.RadioBox
-	container: wx.Panel
-	panel: Union[ClientPanel, ServerPanel]
-	mainSizer: wx.BoxSizer
+class DirectConnectDialog(ContextHelpMixin, wx.Dialog):
+	helpId = "RemoteAccessConnect"
+	_selectedPanel: ClientPanel | ServerPanel
 
-	def __init__(self, parent: wx.Window, id: int, title: str, hostnames: Optional[List[str]] = None):
+	def __init__(self, parent: wx.Window, id: int, title: str, hostnames: list[str] | None = None):
 		super().__init__(parent, id, title=title)
-		mainSizer = self.mainSizer = wx.BoxSizer(wx.VERTICAL)
-		self.clientOrServer = wx.RadioBox(
-			self,
-			wx.ID_ANY,
-			choices=(
-				# Translators: A choice to connect to another machine.
-				_("Client"),
-				# Translators: A choice to allow another machine to connect to this machine.
-				_("Server"),
-			),
-			style=wx.RA_VERTICAL,
+		mainSizer = wx.BoxSizer(wx.VERTICAL)
+		contentsSizerHelper = BoxSizerHelper(self, wx.VERTICAL)
+		self._connectionModeControl = contentsSizerHelper.addLabeledControl(
+			# Translators: Label of the control allowing users to set whether they are the controlling or controlled computer in the Remote Access connection dialog.
+			pgettext("remote", "&Mode"),
```

Comment:
Consistent with the Settings dialog
```suggestion
			pgettext("remote", "&Mode:"),
```